### ISSUE TITLE
Midterm review by bje43

### ISSUE THREAD

**Issue by Benedds** [ft99/ORIE4741#7]
This project seeks to create a tool to predict whether an individual spotify user will like a particular song based upon details about the song and a user's previous history. In order to go about this analysis, the writers manually classified some features of the songs such as genre and whether they liked the song and combined it with techincal information provided by Spotify about the song.

Things that I liked:
	1) I liked the section of future feature transformations and how it addressed the inaccuracies in not using a one hot encoding.
	2) The future work section was detailed and addressed much of what I wanted to see more of from the project. 
	3) The introduction clearly establishes the goal of the project and differentiates it from Spotify's existing tools.

Things that can be improved:
	1) The goal of the project is create a tool that can determine whether an individual user will like a song.
However, when creating predictive models, you combined the like or not data of multiple users. These models wouldn't be able to accurately predict one person's preferences because they have data from other users who are going to have different song preferences. 
	2) I wish there was more description in the second section "Data Collection, Creation and Connection". Why were only four genres chosen and how did the group decide which song was in each genre. Did you guys discuss the definition of the four genres before hand? Manually classifying the songs seems prone to many errors. 
	3) I would have liked more details about the models that were run. How were the test set and training set formed and what percentage of the data was in each?
